lib/types/application/ApplicationBuilder.js

```diff
@@ -61,7 +43,7 @@ class ApplicationBuilder extends AbstractBuilder {
 		});
 
 		this.addTask("generateFlexChangesBundle", () => {
-			const generateFlexChangesBundle = tasks.generateFlexChangesBundle;
+			const generateFlexChangesBundle = taskRepository.getTask("generateFlexChangesBundle").task;
 			return generateFlexChangesBundle({
```

Comment:
[minor][consistency] different contributor, different style? all other tasks are retrieved and configured in a single call chain

Reply:
Possibly, I aligned all to the one-liner.